### ISSUE TITLE
Consider adding recommendations when setting types to Carbon documents

### ISSUE THREAD

**Issue by alex-arriaga** [CarbonLDP/carbonldp-website#215]
In the JavaScript SDK docs, there is this paragraph about the usage of "types"; I think it would be good to list a set of recommendations when adding types to a document, for example in the ES2015 example I see: 

```
// ... imports
let project;

// ... project retrieval

project.types.push( "project", "important-project" );
```
However, we created our types following CamelCase notation, I am not sure if that has any implication in the future, but coming from an Object Oriented mindset, my first reaction is to name my "types" as "classes", e. g. Person, Dog, Product, etc. Is that good? Is there any convention in the Linked Data world that I am missing?

**Comment by MiguelAraCo** [comment]
Indeed, it is recommended to use CamelCase notation. We'll change the examples to reflect this.

**Comment by MiguelAraCo** [comment]
It is important to note that using other naming notations isn't wrong per-se. It just wouldn't be following RDF naming conventions/best practices.
